content/docs/04-guild-data.md

```diff
@@ -1,12 +1,17 @@
-# Guild data
+# Guild Data
 
-The data that we collect is only for our own legitimate interests and we only use it in ways that are explicitly specified in the [Privacy Policy](/privacy).
+At Countr, we take your privacy seriously. The data we collect is only used for our own legitimate interests and is handled in accordance with our [Privacy Policy](/privacy).
 
 
-## How to manage your guild data {#how}
+## Managing Your Guild Data {#how}
 
-You can use the `/data export all` command to extract your guild's data. You can also extract your counting channel's flows individually using the `/data export flow/flows` commands, and also extract your scores table using the `/data export scores` command.
+We understand that you may want to manage your guild's data, and we've made it easy to do so. Here are some commands you can use to manage your data:
 
-You can also import a flow or a scores table using the `/data import flow` and `/data import scores` commands.
+* `/data export all`: Use this command to extract all of your guild's data.
+* `/data export flow/flows`: Use this command to extract your counting channel's flows individually.
+* `/data export scores`: Use this command to extract your scores table.
+* `/data import flow`: Use this command to import a flow.
+* `/data import scores`: Use this command to import a scores table.
+* `/data reset`: Use this command to reset your guild's data.
 
-If you want to reset your guild's data, you can use the `/data reset` command.
\ No newline at end of file
+We hope these commands make it easy for you to manage your data. If you have any questions or concerns, please don't hesitate to reach out to us.
```

Comment:
This just smells like AI to me and I don't love it, maybe just completely reroll this lol